coderd/rbac/roles.go

```diff
@@ -406,6 +432,59 @@ func ReloadBuiltinRoles(opts *RoleOptions) {
 				},
 			}
 		},
+		orgAuditor: func(organizationID uuid.UUID) Role {
+			return Role{
+				Identifier:  RoleIdentifier{Name: orgAuditor, OrganizationID: organizationID},
+				DisplayName: "Organization Auditor",
+				Site:        []Permission{},
+				Org: map[string][]Permission{
+					organizationID.String(): Permissions(map[string][]policy.Action{
+						ResourceAuditLog.Type: {policy.ActionRead},
+					}),
+				},
+				User: []Permission{},
+			}
+		},
+		orgUserAdmin: func(organizationID uuid.UUID) Role {
+			// Manages organization members and groups.
+			return Role{
+				Identifier:  RoleIdentifier{Name: orgUserAdmin, OrganizationID: organizationID},
+				DisplayName: "Organization User Admin",
+				Site: Permissions(map[string][]policy.Action{
+					// To assign organization members, we need to be able to read
+					// users at the site wide to know they exist.
+					ResourceUser.Type: {policy.ActionRead},
+				}),
```

Comment:
This needs to be fixed by only needing the org member read permission. But `ExtractUser` is currently used in organization routes, where an org member should probably be sufficient.